alf/trainers/policy_trainer.py

```diff
@@ -261,7 +261,12 @@ def train(self):
         self._save_checkpoint()
         if self._evaluate:
             self._eval_env.pyenv.close()
-        self._env.pyenv.close()
+        try:
+            # async off-policy driver has several envs, including self._env
+            for env in self._envs:
+                env.pyenv.close()
+        except:
```

Comment:
Never use the `except` without specifying what kind of exception you want to capture. This can even capture spelling errors